flict/flictlib/project/reader.py

```diff
@@ -8,6 +8,8 @@
 
 from flict.flictlib.return_codes import FlictError, ReturnCodes
 
+DEPENDENCY_TAGS = ["DYNAMIC_LINK", "STATIC_LINK", "DEPENDS_ON", "CONTAINS", "COPY_OF"]
```

Comment:
Please refrain from using double quotes - the current style is single ticks only

Reply:
Still open

Reply:
Fixed in coming push